docs/src/gs/create-workload-cluster.md

```diff
@@ -6,41 +6,47 @@ Choose one of the available templates for to create your workload clusters from
-| `OCI_SSH_KEY`                   |                     | The public SSH key to be added to the Kubernetes nodes. It can be used to login to the node and troubleshoot failures.                                                                                                                                                                                                                                                                                    |
-| `OCI_PV_TRANSIT_ENCRYPTION`     | true                | [In-transit encryption](https://docs.oracle.com/en-us/iaas/Content/File/Tasks/intransitencryption.htm) provides a way to secure your data between instances and mounted file systems using TLS v.1.2 (Transport Layer Security) encryption. Only [some bare metal instances](https://docs.oracle.com/en-us/iaas/releasenotes/changes/60d602f5-abb3-4639-aa19-292a5744a808/) support In-transit encryption |
+The following Oracle Cloud Infrastructure (OCI) configuration parameters are available when creating a workload 
+cluster on OCI using one of our predefined templates:
+
+| Parameter                                 | Default Value       | Description                                                                                                                                                                                                                                                                                                                                                                                                  |
+|-------------------------------------------|---------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `OCI_COMPARTMENT_ID`                      |                     | The OCID of the compartment where the OCI resources in which the resources are created                                                                                                                                                                                                                                                                                                                       |
+| `OCI_IMAGE_ID`                            |                     | The OCID of the image (Oracle Linux or <br/>Ubuntu) with which to create the Kubernetes nodes. This same image is used for both the control plane and the worker nodes.                                                                                                                                                                                                                                      |
```

Comment:
Dont think we need to specify Ol /Ubunt etc, just say image of kubernetes nodes. We may add more types later and we don't need to change the deocs every time.